build.c

```diff
@@ -443,6 +443,8 @@ edgedone(struct edge *e)
 	bool restat, prune, pruned;
 	int64_t old;
 
+	/* mark edge clean for dyndepload */
+	e->flags &= ~FLAG_DIRTY;
```

Comment:
I think this might break manifest rebuilds. We currently use the dirty flag to check if we actually needed to rebuild it (see #31).